### PR TITLE
export docs in data.core

### PR DESCRIPTION
**Pull request by KushajveerSingh**
Exported docs of TfmdDL and TfmdLists to the library.

### CODE REVIEW

File: fastai/data/core.py [hunk 1]
```diff
@@ -269,6 +279,17 @@ def __getitem__(self, idx):
         if self._after_item is None: return res
         return self._after_item(res) if is_indexer(idx) else res.map(self._after_item)
 
+# Cell
+add_docs(TfmdLists,
+         setup="Transform setup with self",
+         decode="From `Pipeline",
```
Comment (jph00) [inline fastai/data/core.py:285]: Looks like two lines are missing closing backticks here

**Review by jph00 — CHANGES_REQUESTED** [review]
Many thanks! Just one change requested

File: fastai/data/core.py (hunk 1 above)
Comment (KushajveerSingh) [reply fastai/data/core.py]: Done.

**Comment by KushajveerSingh** [comment]
Done.

**Pull request merged**